templates/corporate/support/realm_details.html

```diff
@@ -38,6 +38,17 @@ <h3><img src="{{ realm_icon_url(realm) }}" class="support-realm-icon" /> {{ real
     <b>First human user</b>:
     <br />
     {% endif %}
+    {% set billing_admins = realm.get_billing_admins_delivery_email() %}
+    {% if billing_admins %}
+    <b>Billing admins</b>: {{ billing_admins  }}
+    <a title="Copy emails" class="copy-button" data-clipboard-text="{{ billing_admins.delivery_email }}">
```

Comment:
How can a `str` have a `delivery_email` property?

Reply:
Fixed, thanks for catching! Tested that the copy is working as expected with the pushed changes.